include/libopencm3/stm32/common/i2c_common_v1.h

```diff
@@ -365,6 +371,23 @@ specific memorymap.h header before including this header file.*/
  * TRISE[5:0]: Maximum rise time in Fast/Standard mode (master mode)
  */
 
+/* --- I2Cx_FLTR values ----------------------------------------------------- */
+
+/*
```

Comment:
if you're adding docs, please add them as doxygen, and check they look how you expect from "make doc"

Reply:
I was matching style for style in the existing file.  Could use your guidance on if a commit that both doxygenizes and changes a file is "mixed changes in a single commit" or not. Happy to get the register issue first, and then go back and doxygenize and submit as a second PR, or change it into two commits in this PR. Let me know what you prefer.

Reply:
if the old style is non-doxygen, there's not a lot of reason to follow it.  You don't have to fix anything pre-existing, though, and yes, this might make a file mixed.